opal/hwloc/hwloc-internal.h

```diff
@@ -190,8 +192,6 @@ OPAL_DECLSPEC extern bool opal_hwloc_use_hwthreads_as_cpus;
 #define HWLOC_OBJ_L3CACHE HWLOC_OBJ_CACHE
 #define HWLOC_OBJ_L2CACHE HWLOC_OBJ_CACHE
 #define HWLOC_OBJ_L1CACHE HWLOC_OBJ_CACHE
-#define HWLOC_OBJ_NUMANODE HWLOC_OBJ_NODE
-#define HWLOC_OBJ_PACKAGE HWLOC_OBJ_SOCKET
```

Comment:
I believe you will still need these as hwloc 1.11.x doesn't know the numanode and package values